ocr_service/tests/api/test_ocr_endpoints.py

```
@@ -0,0 +1,17 @@
+from fastapi.testclient import TestClient
+from main import app
+
+client = TestClient(app)
+
+def test_root_endpoint():
+    """Test the root endpoint returns the expected message"""
+    response = client.get("/")
+    assert response.status_code == 200
+    assert response.json() == {"message": "Medical Document OCR Service is online"}
+
+def test_health_check():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_health_check`

Add return type annotation: `None`

(ANN201)